source/configure/experimental-configuration-settings.rst

```diff
@@ -387,6 +387,21 @@ This setting enables the Apps Bar and moves all Mattermost integration icons fro
 | This feature's ``config.json`` setting is ``"EnableAppBar": false`` with options ``true`` and ``false``. |
 +----------------------------------------------------------------------------------------------------------+
 
+Patch React DOM used by plugins
```

Comment:
This may also need to be documented at https://docs.mattermost.com/manage/telemetry.html.